tox.ini

```diff
@@ -18,7 +22,9 @@ deps =
 commands =
     pylint edxval
     pylint --py3k edxval
-    pycodestyle edxval
-    pydocstyle edxval
-    isort --check-only --recursive edxval manage.py setup.py
+    pycodestyle edxval --ignore=D100,D101,D102,D104,D105,D106,D107,D200,D202,D203,D204,D205,D210,D212,D213,D215,D400,D401,D403,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414
+    pydocstyle edxval --ignore=D100,D101,D102,D104,D105,D106,D107,D200,D202,D203,D204,D205,D210,D212,D213,D215,D400,D401,D403,D404,D405,D406,D407,D408,D409,D410,D411,D412,D413,D414
```

Comment:
Could these ignores go in the appropriate config sections with some explanation, like in https://github.com/edx/edx-platform/blob/master/setup.cfg#L16-L33 ?